.github/workflows/pathogen-repo-build.yaml

```diff
@@ -0,0 +1,263 @@
+        uses: ./actions/setup-nextstrain-cli
+        with:
+          runtime: ${{ inputs.runtime }}
+        # Required to set up aws-batch as default runtime
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      - name: Checkout build repository
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.repo }}
+          path: ${{ env.BUILD_DIR }}
+
+      - if: inputs.env
+        name: Set environment variables
+        env:
+          env: ${{ inputs.env }}
+        run: >
+          # shellcheck disable=SC2154
+
+          echo "$env"
+          | ./bin/yaml-to-envvars
+          | tee -a "$GITHUB_ENV"
+
+      - if: inputs.runtime-envvar-names
+        name: Set secret runtime environment variables
+        env:
+          secrets: ${{ toJson(secrets) }}
+          varnames: ${{ inputs.runtime-envvar-names }}
+        run: >
+          # shellcheck disable=SC2154
+
+          echo "$secrets"
+          | ./bin/json-to-envvars --varnames="$varnames"
+          | tee -a "$GITHUB_ENV"
+
+      - name: Set build environment variables
+        env:
+          varnames: ${{ inputs.runtime-envvar-names }}
+        run: |
+          # shellcheck disable=SC2154
+
+          # read exits with 1 when it encounters EOF
+          read -a varnames -r -d $'\0' <<<"$varnames" || true
+          ./bin/write-envdir "$BUILD_DIR/$ENV_DIR" "${varnames[@]}"
+
+      - name: Run build via ${{ inputs.runtime }}
+        env:
+          cpus: ${{ inputs.cpus }}
+          memory: ${{ inputs.memory }}
+          detach: ${{ inputs.runtime == 'aws-batch' || null }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        run: |
+          # shellcheck disable=SC2154
+          set -x
+
+          nextstrain build \
+            --cpus "$cpus" \
+            ${memory:+--memory="$memory"} \
+            ${detach:+--detach} \
+            --exec env \
+            "$BUILD_DIR" \
+              envdir "$ENV_DIR" snakemake \
+              --cores "$cpus" \
+              ${{ inputs.build-args }} \
+          | tee build.log
+
+      - if: ${{ inputs.runtime == 'aws-batch' }}
+        name: Get AWS Batch job id
+        id: aws-batch
+        run: |
+          echo "AWS_BATCH_JOB_ID=$(tail -n1 build.log | sed -nE 's/.+attach ([-a-f0-9]+).+/\1/p')" >> "$GITHUB_ENV"
+
+      - if: env.AWS_BATCH_JOB_ID
+        name: Generate AWS Batch summary
+        run: |
+          ./bin/interpolate-env < ./text-templates/attach-aws-batch.md \
+          > "$GITHUB_STEP_SUMMARY"
+
+      - if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          if-no-files-found: warn
+          name: ${{ inputs.artifact-name }}
+          path: |
+            build.log
+            ${{ inputs.artifact-paths }}
+            !${{ env.ENV_DIR }}
```

Comment:
OH, the artifact-paths don't work as-is because they need to be prefixed with `BUILD_DIR/`

Reply:
Added the `BUILD_DIR` prefix in https://github.com/nextstrain/.github/pull/44/commits/d1568bc75c9f19f2fdcb2de4b3815d3aa988a986